doc/api/dns.md

```diff
@@ -484,10 +496,20 @@ added: v0.11.3
 -->
 - `servers` {string[]}
 
-Sets the IP addresses of the servers to be used when resolving. The `servers`
-argument is an array of IPv4 or IPv6 addresses.
+Sets the IP addresses and port of the servers to be used when resolving. The
```

Comment:
```
Sets the IP address and port of servers to be used when performing DNS
resolution. If the port is the IANA default DNS port (53) it can be omitted.
```